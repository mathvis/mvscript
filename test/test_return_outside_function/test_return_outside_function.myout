error: A return statement must be inside a block.
 --> test/test_return_outside_function/test_return_outside_function.mvs:1:1
  |
1 | return 0;
  | ^
  |
hint: Try putting this statement inside of a block or function.
