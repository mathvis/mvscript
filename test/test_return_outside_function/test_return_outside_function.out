error: A return statement must be inside a function.
 --> test/test_return_outside_function/test_return_outside_function.mvs:1:1
  |
1 | return 0;
  | ^
  |
hint: Try putting this statement inside of a function.
